addons/auth_totp/views/user_preferences.xml

```diff
@@ -110,33 +136,80 @@
-                                Two-factor authentication not enabled
-                                <a href="https://www.odoo.com/documentation/14.0/applications/general/auth/2fa.html"
-                                   title="What is this?" class="o_doc_link" target="_blank"></a>
-                            </span>
-                            <button name="totp_enable_wizard" type="object" string="Enable two-factor authentication"
-                                    class="btn btn-info mx-3"/>
-                        </div>
-                    </group>
-                    <group attrs="{'invisible': [('totp_enabled', '=', False)]}">
-                        <div>
-                            <span class="text-success">
-                                <i class="fa fa-check-circle"/>
-                                Two-factor authentication enabled
-                                <a href="https://www.odoo.com/documentation/14.0/applications/general/auth/2fa.html"
-                                   title="What is this?" class="o_doc_link" target="_blank"></a>
-                            </span>
-                            <button name="totp_disable" type="object" string="(Disable two-factor authentication)"
-                                    class="btn btn-link text-muted"/>
+            <group name="auth" position="after">
+                <field name="totp_enabled" invisible="1"/>
+                <group>
+                    <div>
+                        <div class="o_horizontal_separator mt-0">2-Step Authentication
+                            <button attrs="{'invisible': [('totp_enabled', '!=', False)]}" name="totp_enable_wizard" type="object"
+                                    class="fa fa-toggle-on o_auth_2fa_btn disabled" aria-label="Enable TOTP"/>
+                            <button attrs="{'invisible': [('totp_enabled', '=', False)]}" name="totp_disable" type="object"
+                                    class="fa fa-toggle-on o_auth_2fa_btn enabled" aria-label="Disable TOTP"/>
                         </div>
-                    </group>
-            </button>
+                        <span attrs="{'invisible': [('totp_enabled', '!=', False)]}" class="text-muted">
+                            2-Step Authentication ("2FA") is a system of double authentication. The first one is your usual password and the second one is a code you get from a phone app or a sms. Odoo advises you to install a 2FA phone-based authenticator like Authy on your phone.
+                        <a href="https://www.odoo.com/documentation/14.0/applications/general/auth/2fa.html"
+                           title="Learn More" target="_blank">Learn More</a>
+                    </span>
+                    </div>
+                </group>
+            </group>
+        </field>
+    </record>
+
+    <!-- Invite to user 2FA -->
+    <record model="ir.actions.server" id="action_invite_totp">
```

Comment:
To be defined in a ``data/ir_actions_data.xml`` file (and if you are motivated, existing one could be moved as they don't belong to views)